### ISSUE TITLE
Bug: Script Fails When `dde` Network is Missing

### ISSUE THREAD

**Issue by sbaerlocher** [whatwedo/dde#70]
**Description**
When executing the `dde system:update` command, an error occurs if the `dde` network is not present. This results in an error message and the termination of the script. The exact error is: `Error response from daemon: network dde not found`.

**Steps to Reproduce**
1. Run the `dde system:update` command without the `dde` network being present.
2. Observe the error.

**Expected Behavior**
The script should handle the absence of the `dde` network gracefully and display an appropriate message instead of failing with an error.

**Actual Behavior**
The script crashes with the error message `Error response from daemon: network dde not found`.

**Possible Solution**
Add a check in line 11 of the `destroy.sh` script to verify if the `dde` network exists before attempting to remove it. This could be achieved with the following bash code:

```bash
if docker network ls | grep -q dde; then
  docker network rm dde
else
  echo "Network 'dde' not found, no removal necessary."
fi

**Comment by rugbymauri** [comment]
not sure, but the problem is caused by the `"` in the docker-compose.yaml


this should work

```
networks:
    default:
        name: dde
        external: true

```

**Comment by sbaerlocher** [comment]
No, I do not suspect that this is the problem, as executing `dde system:destroy` directly works without any issues. However, when combined – first executing `dde system:destroy` and then `dde system:update` – it leads to network problems between lines 10-12 in the destroy.sh, during the execution of the command from `dde system:update`.
